firestore/firestore.rules

```diff
@@ -25,7 +25,17 @@
       return get(/databases/$(database)/documents/surveys/$(surveyId)).data;
     }
 
-   /**
+    /**
+     * Returns true iff the survey doesn't have a restricted access.
```

Comment:
`Returns true iff all authenticated users can read and contribute data to the specified survey.`